src/shared/install.c

```diff
@@ -1840,7 +1839,7 @@ int unit_file_mask(
                 if (!path)
                         return -ENOMEM;
 
-                q = create_symlink(&paths, "/dev/null", path, force, changes, n_changes);
+                q = create_symlink(&paths, "/dev/null", path, flags & UNIT_FILE_FORCE, changes, n_changes);
```

Comment:
this a teensy bit ugly, as you potentially pass something that is > 1 here in a bool. Which is actually Ok, but still makes me stumble a bit. I'd probably enclose this in `!!()`, so that it becomes `!!(flags & UNIT_FILE_FORCE)`...